app/javascript/src/components/Dashboard/Notes/NotesList.jsx

```diff
@@ -41,7 +40,7 @@ const NotesList = ({ key, note, setSelectedNoteIds, setShowDeleteAlert }) => {
             position="bottom"
           >
             <Typography style="body3">
-              Created {calculateCreatedAgo(note.created_at)}{" "}
+              Created {`${calculateCreatedAgo(note.created_at)} `}
```

Comment:
Put this whole message in a template string. Right now, the importance of the whitespace after the word `Created` is not very well signified in the code.

PS: This doesn't mean we should use template strings all the time. Unless there is string interpolation, don't use template strings.

Reply:
Ok @yedhink.
Can you please check if the code below is the right solution :-
 {`Created ${calculateCreatedAgo(note.created_at)} `}

Reply:
No. Put the whole string in the template string.

Read: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

```suggestion
              {`Created ${calculateCreatedAgo(note.created_at)} `}
```

Reply:
Done. 
Thank you for reviewing.